### PR TITLE
Feat/cl vault proptest assert

### PR DESCRIPTION
**Pull request by magiodev**
## 4. Checklist

- [ ] Does the Readme need to be updated?

### CODE REVIEW

**Review by LaurensKubat — APPROVED** [review]
1 small comment for the ci

File: smart-contracts/contracts/cl-vault/src/rewards/helpers.rs [hunk 1]
```diff
@@ -89,7 +89,7 @@ impl Rewards {
     }
 
     pub fn claim(&mut self, recipient: &str) -> ContractResult<CosmosMsg> {
-        let rewards = self.coins();
+        let rewards = sort_tokens(self.coins());
```
[the remark this answers is not in the document]
Comment (magiodev) [reply smart-contracts/contracts/cl-vault/src/rewards/helpers.rs:92]: If you remove this the proptest fails and you cant even debug the first line of the execute_exact_deposit(). May you try please just for sanity check?

[the remark this answers is not in the document]
Comment (LaurensKubat) [reply smart-contracts/contracts/cl-vault/src/rewards/helpers.rs:92]: sure

**Review by LaurensKubat — APPROVED** [review]
lgtm

**Pull request merged**